appcues/src/main/java/com/appcues/ui/utils/AppcuesMargins.kt

```diff
@@ -60,3 +63,53 @@ private class MarginValuesModifier(
         return paddingValues == otherModifier.paddingValues
     }
 }
+
+// This is a copy/paste of the standard PaddingValuesImpl from compose foundation, but without
+// the restriction against negative values, as we allow negative margins and handle that above
+// in our MarginValuesModifier. Prior to Compose 1.7 we still used the standard PaddingValuesImpl,
+// but in Compose 1.7 they added checks that throw exceptions if negative values are used, so
+// we need our own.
+//
+// This implementation still implements the same PaddingValues interface, so there is minimal impact to
+// existing call sites.
+//
+// the change to check negative values was in this update:
+// https://android-review.googlesource.com/c/platform/frameworks/support/+/3021136/3/compose/foundation/foundation-layout/src/commonMain/kotlin/androidx/compose/foundation/layout/Padding.kt
+@Stable
+@Suppress("FunctionNaming") // matching Compose naming for PaddingValues here
+internal fun MarginValues(all: Dp): PaddingValues = MarginValues(all, all, all, all)
```

Comment:
I believe this is the way to go here too, we had this done for the Modifier too, not sure why they felt the need to add this restriction to the PaddingValues implementation as well